src/qt/overviewpage.cpp

```diff
@@ -402,7 +403,10 @@ void OverviewPage::updateCoinJoinProgress()
 
     if(nMaxToAnonymize == 0) return;
 
-    if (nMaxToAnonymize >= clientModel->coinJoinOptions().getAmount() * COIN) {
+    if (m_privacy) {
+        strAmountAndRounds = "#### DASH / ## Rounds";
```

Comment:
I think it's possible to use here `BitcoinUnits::name` to show proper name when required (tDASH, mDash, etc)